### PR TITLE
LPS-70161

### PR DESCRIPTION
**Pull request by hhuijser**

### CODE REVIEW

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: portal-web/docroot/html/taglib/ui/input_time_zone/page.jsp [hunk 1]
```diff
@@ -90,7 +90,7 @@ numberFormat.setMinimumIntegerDigits(2);
 
 			TimeZoneFormat icuTimeZoneFormat = icuSimpleDateFormat.getTimeZoneFormat();
 
-			sb.append(icuTimeZoneFormat.format(TimeZoneFormat.Style.GENERIC_LOCATION, icuTimeZone, date.getTime()));
+			sb.append(icuTimeZoneFormat.format(TimeZoneFormat.Style.ZONE_ID, icuTimeZone, date.getTime()));
```
Comment (drewbrokke) [inline portal-web/docroot/html/taglib/ui/input_time_zone/page.jsp]: @daledotshan @hhuijser 

A side effect of this change is that when the portal language is Japanese, the entry displays English for the appended part:

**Before fix:**
`(UTC -07:00) 山地標準時 (フェニックス時間)`

**After fix:**
`(UTC -07:00) 山地標準時 (America/Phoenix)`

According to the ticket, this fix will result in the correct Expected/Actual behavior, but step #7 is no longer correct.  Is this intended?

Comment (daledotshan) [inline portal-web/docroot/html/taglib/ui/input_time_zone/page.jsp]: Hey @drewbrokke 
Before answer your question I would like to explain the issue again.
If we set language to Japanese for operating system and set portal language to English you will got following behavior
 (UTC -07:00) Mountain stand time (フェニックス時間)
Actually it always show Japanese no matter what language portal set as it read the timezone information from operating system level

After I changed the code I would like it always show as English no matter what language portal set (it is just a timezone id)


Please let me know if you have better way to fix this issue
Thanks,
Dale

Comment (drewbrokke) [inline portal-web/docroot/html/taglib/ui/input_time_zone/page.jsp]: @daledotshan thanks for the explanation.  That makes sense.  I did some re-factoring (un-related), but your changes work well. Thank you!

**Comment by drewbrokke** [comment]
Pull request sent to https://github.com/pei-jung/liferay-portal/pull/177.  Thanks!

**Pull request closed without merging**